x-pack/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -132,9 +132,9 @@ export function DetailView({ errorGroup, urlParams, location }: Props) {
       label: i18n.translate('xpack.apm.errorGroupDetails.handledLabel', {
         defaultMessage: 'Handled'
       }),
-      val:
-        String(idx(error, _ => _.error.exception[0].handled)) ||
-        NOT_AVAILABLE_LABEL,
+      val: String(
+        idx(error, _ => _.error.exception[0].handled) || NOT_AVAILABLE_LABEL
+      ),
```

Comment:
As it is right now, this change would only display 'true' or 'N/A' but i think we still want to display 'false' if handled is set to `false`

Reply:
Oh, you are right! I'll have to check if it's a boolean first.